pom.xml

```diff
@@ -252,7 +270,7 @@
                     <!-- JLINK -->
                     <plugin>
                         <artifactId>maven-jlink-plugin</artifactId>
-                        <version>3.0.0-alpha-1</version>
+                        <version>3.0.0-alpha-2-SNAPSHOT</version>
```

Comment:
Quick question: I did see this earlier -- I assume there's something that earlier alpha is missing that is needed?

Reply:
There's a bug in the first alpha that shows its head in this build - Where NullPointerException is thrown during the dependency finding, 
It is NB to move to alpha 2, I also included the snapshot repository for plugins :)